pandas/tests/frame/methods/test_append.py

```diff
@@ -96,6 +96,18 @@ def test_append_list_of_series_dicts(self):
         expected = df.append(DataFrame(dicts), ignore_index=True, sort=True)
         tm.assert_frame_equal(result, expected)
 
+    def test_append_list_retain_index_name(self):
+        df = DataFrame(
+            [[1, 2], [3, 4]], index=pd.Index(["a", "b"], name="keepthisname")
+        )
+
+        serc = Series([5, 6], name="c")
+
+        result = df.append([serc])
+        expected = df.append(serc)
```

Comment:
can you explicitly construct the expected here. also would parameterize this test on both `[serc]` and `serc` in the append.

Reply:
Hi @jreback, Thanks for your feedback. I guess that with your second sentence you mean to perform assert_frame_equal for append on `[serc]` and `serc` separetely?

Reply:
yep exactly